src/pages/App.tsx

```diff
@@ -184,124 +192,141 @@ export default function App() {
 
   const isHeaderTransparent = !scrolledState
 
+  const statsigUser = useMemo(
+    () => ({
+      userID: getDeviceId(),
+    }),
+    []
+  )
+
   return (
     <ErrorBoundary>
       <DarkModeQueryParamReader />
       <ApeModeQueryParamReader />
       <Trace page={currentPage}>
-        <HeaderWrapper transparent={isHeaderTransparent}>
-          <NavBar />
-        </HeaderWrapper>
-        <BodyWrapper>
-          <Popups />
-          <Polling />
-          <TopLevelModals />
-          <Suspense fallback={<Loader />}>
-            {isLoaded ? (
-              <Routes>
-                <Route path="/" element={<Landing />} />
+        <StatsigProvider
+          user={statsigUser}
+          // TODO: replace with proxy and cycle key
+          sdkKey="client-1rY92WZGidd2hgW4x1lsZ7afqm1Qfr3sJfH3A5b8eJa"
+          waitForInitialization={true}
+          options={{
+            environment: { tier: getEnvName() },
+          }}
+        >
```

Comment:
ended up moving the provider here because the Amplitude Device ID isn't available at first render